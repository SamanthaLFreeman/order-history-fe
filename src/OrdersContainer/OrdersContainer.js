import React from 'react';
import './OrdersContainer.css';

const OrdersContainer = ({orders}) => {
  return (
    <section>
      hey
    </section>
  )
}

export default OrdersContainer;