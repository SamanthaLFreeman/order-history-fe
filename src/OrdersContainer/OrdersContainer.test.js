import React from 'react';
import { shallow } from 'enzyme';
import OrdersContainer from './OrdersContainer';

describe('OrdersContainer', () => {
  let wrapper;

  beforeEach(() => {
    const mockOrders = [
      {
        name: 'Jeans',
        img: 'fancy looking jeans',
        description: 'Obviously pants...',
        price: '10000000',
        id: 42
    }
  ]
    wrapper = shallow(<OrdersContainer orders={mockOrders} removeOrder={jest.fn()}/>)
  })

  it('should match the snapshot with data passed through', () => {
    expect(wrapper).toMatchSnapshot();
  })

  it('should match the snapshot with no data passed through', () => {
    const mockOrders = [
      {
        name: '',
        img: '',
        description: '',
        price: '',
        id: ''
      }
    ]
    const wrapper = shallow(<OrdersContainer orders={mockOrders} removeOrder={jest.fn()} />)
    expect(wrapper).toMatchSnapshot();
  })
})