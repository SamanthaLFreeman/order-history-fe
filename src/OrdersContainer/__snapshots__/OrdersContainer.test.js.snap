// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OrdersContainer should match the snapshot with data passed through 1`] = `
<section
  className="orders-container"
>
  <Order
    key="42"
    order={
      Object {
        "description": "Obviously pants...",
        "id": 42,
        "img": "fancy looking jeans",
        "name": "Jeans",
        "price": "10000000",
      }
    }
    removeOrder={[MockFunction]}
  />
</section>
`;

exports[`OrdersContainer should match the snapshot with no data passed through 1`] = `
<section
  className="orders-container"
>
  <Order
    key=""
    order={
      Object {
        "description": "",
        "id": "",
        "img": "",
        "name": "",
        "price": "",
      }
    }
    removeOrder={[MockFunction]}
  />
</section>
`;
